app/lambda/lambda_function.py

```diff
@@ -1,28 +1,57 @@
+# import boto3
```

Comment:
Please remove out commented code

Reply:
Done.

Reply:
Added in another comment on the new PR
https://github.com/aws-samples/medical-transcription-analysis/pull/21/files